packages/components/src/components/Button/index.tsx

```diff
@@ -108,10 +111,14 @@ export interface ButtonProps
   to?: string;
   as?: any;
   target?: any;
+  autoWidth?: boolean;
```

Comment:
can we call this prop `inline` which does the opposite of `block` in `Text` = 

display: inline-block
width: fit-content

Reply:
But it doesn't do that, it doesn't set to inline :(

Reply:
It should though? I imagine the intent here isn't just to make it width: auto but also for it to not take the whole block of space (display: block) and instead either take only as much space needed or space decided by it's parent (sounds like inline-block)

Reply:
Actually no, I literally just don't want it to span 100%

Most of these don't have any other thing set
![CleanShot 2020-06-03 at 11 29 21](https://user-images.githubusercontent.com/1051509/83620343-7af3ef00-a58d-11ea-9911-e22051acc067.png)

Reply:
In my head, those are two different things, I want them block but not 100%

Reply:
I can call it small?

Reply:
We decided to keep it for now